admin/data/sql/migrations/20190402121931.do.add-unique-undex-pupil-table-upn-and-school-id.sql

```diff
@@ -0,0 +1,6 @@
+DROP INDEX [mtc_admin].pupil.pupil_upn_uindex;
+CREATE UNIQUE NONCLUSTERED INDEX [pupil_upn_uindex] ON [mtc_admin].[pupil]
```

Comment:
The  index name must include the school_id component too.

Reply:
@jon-shipley added.